packages/playwright/test/RunPartial/OptionsTests.cs

```diff
@@ -0,0 +1,61 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Linq;
+using Deque.AxeCore.Commons;
+using NUnit.Framework;
+
+namespace Deque.AxeCore.Playwright.Test.RunPartial
+{
+    public class OptionsTests : TestBase
+    {
+        protected const string RULE_TO_TEST = "region";
+
+        [Test]
+        public async Task ShouldPassOptionsToAxeCore()
+        {
+            await GoToFixture("index.html");
+
+            var rules = new Dictionary<string, RuleOptions>();
+            rules.Add(RULE_TO_TEST, new RuleOptions
+            {
+                Enabled = true
+            });
+
+            var results = await Page!.RunAxe(new AxeRunOptions
+            {
+                Rules = rules
+            });
+
+            var combinedRules = new List<AxeResultItem>();
+            combinedRules.AddRange(results.Passes);
+            combinedRules.AddRange(results.Inapplicable);
+            combinedRules.AddRange(results.Incomplete);
+            combinedRules.AddRange(results.Violations);
+
+            Assert.IsTrue(combinedRules.Any(x => x.Id == RULE_TO_TEST));
```

Comment:
I'm not sure this test works as expected. The region rule is already enabled by default so this isn't verifying that the option was passed. It could completely fail to pass the option and this would still return true. I think instead of `rules` you should pass `runOnly` which would only run the 1 rule, then you could assert that all results are for the rule and no others.